agent.readme.md.backup

```diff
@@ -0,0 +1,117 @@
+# 🤖 agent.readme.md — Refs Project
```

Comment:
We should delete this, I think this was probably automatically generated by the editor